Documentation/feature-band-source-building.md

```diff
@@ -0,0 +1,893 @@
+- Only 1xx and one other band (2xx, 3xx, or 4xx) are in support at any given
+  time.
+- The initial release of each feature band depends on the latest release of
+  the previous band.
+- 3xx and 4xx bands follow identical build patterns; the scenarios documented
+  for 3xx apply equally to 4xx with adjusted version numbers.
+
+## Build Command Arguments
+
+Feature band builds make use of these relevant command-line arguments:
+
+- `--with-packages` - Provides PSB artifacts
+- `--with-shared-components` - Provides shared component artifacts from 1xx build
+- `--with-sdk` - Specifies the SDK to use for building
+
+## Distro Maintainer Workflows
+
+The following sections describe the workflows for different scenarios.
+
+**Note on 4xx band**: The 4xx band follows the same build patterns as the 3xx
+band documented below, with the version numbers adjusted accordingly (e.g.,
+N.0.400 instead of N.0.300). For the initial 4xx release, the dependencies
+would come from the latest 3xx release instead of 2xx. The 3xx band also follows
+the same pattern as 2xx but is documented here for clarity to illustrate that
+later feature bands continue to use 1xx as input, not just the previous band.
+
+### Input Artifacts Summary
+
+**For 1xx band builds:**
+
+- **[Bootstrap](#1xx-band-bootstrap)**:
+  - PSB artifacts: Microsoft-built previous N.0.1xx release
+  - SDK: Microsoft-built previous N.0.1xx release
+- **[Servicing](#1xx-band-servicing)**:
+  - PSB artifacts: Source-built previous N.0.1xx release
+  - SDK: Source-built previous N.0.1xx release
+
+**For 2xx band builds:**
+
+- **[Initial release (N.0.200)](#2xx-band-initial-release-n0200)**:
+  - Shared component artifacts: Source-built current N.0.1xx release
+  - PSB artifacts: Source-built previous N.0.1xx release
+  - SDK: Source-built previous N.0.1xx release
+- **[Bootstrap initial release (N.0.200)](#2xx-band-bootstrap-n0200)**:
+  - Shared component artifacts: Source-built current N.0.1xx release
+  - PSB artifacts: Microsoft-built previous N.0.1xx release
+  - SDK: Microsoft-built previous N.0.1xx release
+- **[Bootstrap (N.0.201+)](#2xx-band-bootstrap-n0200)**:
+  - Shared component artifacts: Source-built current N.0.1xx release
+  - PSB artifacts: Microsoft-built previous N.0.2xx release
+  - SDK: Microsoft-built previous N.0.2xx release
+- **[Servicing (N.0.201+)](#2xx-band-servicing-n0201)**:
+  - Shared component artifacts: Source-built current N.0.1xx release
+  - PSB artifacts: Source-built previous N.0.2xx release
+  - SDK: Source-built previous N.0.2xx release
+
+**For 3xx band builds (4xx follows the same pattern with adjusted version numbers):**
+
+- **[Initial release (N.0.300)](#3xx-band-initial-release-n0300)**:
+  - Shared component artifacts: Source-built current N.0.1xx release
+  - PSB artifacts: Source-built previous N.0.2xx release
+  - SDK: Source-built previous N.0.2xx release
+- **[Bootstrap initial release (N.0.300)](#3xx-band-bootstrap-n0300)**:
```

Comment:
I think splitting out the initial and servicing bootstrap scenarios is not necessary and may hurt the readability especially since they map to the same command steps.  I lean towards abstracting the description a bit and only having a single bootstrap scenario.